static/src/containers/App/index.js

```diff
@@ -21,8 +21,8 @@ class App extends React.Component { // eslint-disable-line react/prefer-stateles
                 <section>
                     <Header />
                     <div
-                      className="container"
-                      style={{ marginTop: 10, paddingBottom: 250 }}
+                      className="container-fluid"
+                      style={{ marginTop: 10, padding: 2 }}
```

Comment:
Is this meant to be 2?

Reply:
Yes it is. - So that "End Game" button can be close to the edge of the screen and that there is no unusable space between the 'main screen' and the footer

Reply:
Cool just making sure it wasn't a typo :)